package.json

```diff
@@ -15,5 +15,8 @@
     "url": "git+https://github.com/jskuby/react-native-gridview.git"
   },
   "author": "Jason Skuby",
-  "license": "ISC"
+  "license": "ISC",
+  "dependencies": {
+    "prop-types": "^15.5.7"
```

Comment:
I chose `^15.5.7` as the version here because it's [what was recommended in the documentation](https://www.npmjs.com/package/prop-types#how-to-depend-on-this-package). Apparently "there are known issues in versions before 15.5.7".